### DISCUSSION TITLE
Tagesertrag mit mehreren WR

### DISCUSSION THREAD

**Discussion by KEausW** [hoylabs/OpenDTU-OnBattery#1444] [Q&A]
Hallo kann mir mal bitte jemand helfen ?
Ich habe zwei WR ein HMS1800 mit vier Platten
und einen HMS1000 mit zwei Platten und einem Marstek B2500
davor.  Der 1800er soll den Tagesbedarf decken und der
1000er geht gegen Abend mit Akkuleistng in Betrieb.
Leider ist es mir bisher nicht gelungen die DTUoB so einzustellen
das der Tagesertrag von beiden WR summiert angezeigt wird.
Sowie der einer den Betrieb mangels Sonne einstellt nullt
sich der Tagesertrag.  Die Einstellungen im Bereich Wechselrichter
habe ich bereits alle duchgetestet, aber Leider ohne Erfolg.
Ist das ein BUG oder hat jemand einen Tip für mich ?

MfG  Karsten

**Comment by schlimmchen** [comment]
> Leider ist es mir bisher nicht gelungen die DTUoB so einzustellen das der Tagesertrag von beiden WR summiert angezeigt wird.

Ich bin verwirrt... Da gibt es nichts einzustellen, der Tagesertrag oben in den gründen Kacheln im Live View, der ist summiert über alle Wechselrichter.

> Sowie der einer den Betrieb mangels Sonne einstellt nullt sich der Tagesertrag.

Hängt laut Beschreibung nicht einer der WR an einem Akku?

Also ich begreife das Setup nicht. Kannst du das vielleicht mal aufmalen?

Und hast du

![image](https://github.com/user-attachments/assets/1b70371b-2e6b-4310-82cb-2000703166b7)

auch schon ausprobiert?

**Comment by KEausW** [comment]
Hallo , hier ist erst mal meine Topografie zum besseren Verständnis. Durch Zufall sah ich gerade auf das Display
als die DTU kurz  "Offline" ging und danach war der Tagesertrag auf 0 gesetzt. Unter Restgrund steht  "Software reset CPU"  
und die Betriebszeit hat ebenfalls bei 0 neu begonnen zu zählen. Ich habe mehrere 
DTUs mit ESP32 4MB getestet deshalb dachte ich das die evtl. zu Schwachbrüstig sind um mit zwei WR und DPL umzugehen .
Deshalb habe ich ein Fusion-Board mit ESP32 S3 gekauft aber das zeigt genau die selben Symptome.
![topo](https://github.com/user-attachments/assets/5976e016-2c73-42b5-a3e9-7f0546c4a9e7)

Das mit Tagesertragskorrektur habe ich alles schon getestet.

**Reply by spcqike** [reply]
Hast du schon mal ein anderes Netzteil am esp probiert? Vielleicht hast du da einfach ein spannungsproblem, was die DTU abstürzen lässt.

**Comment by KEausW** [comment]
Ja ich habe mehrere Netzteile getestet. Das war auch meine erste Vermutung. 
Ich habe gerade gesehen das hier ein Nutzer #1430 auch ein Restart-Problem hat.
Nun habe ich mal geschaut und die DTU hat tatsächlich  zu dieser Zeit ein Zeitabgleich gemacht.
Vielleicht hängt das ja irgendwie zusammen.

**Comment by Manos1966** [comment]
>  die DTUoB so einzustellen
das der Tagesertrag von beiden WR summiert angezeigt wird.
Sowie der einer den Betrieb mangels Sonne einstellt nullt
sich der Tagesertrag. 

"Made bei Design"?

Teste bitte folgendes:

- Was ist die Definition des "Tages" bzw "Nachts"?
(ich kann mich erinnern, "Nacht" ist wenn die Hoymiles Eingaenge < xx W haben)
Also, wenn der HMS-1800 Paneele keinen Strom liefern, die DTU erkennt "Nacht" 

 - Wann fängt der nächste Tag? 
Antwort: wenn die Hoymiles Paneele wieder Strom produzieren.
 - Wann passiert das? 
Antwort: Wenn der HM-1000 anfaengt Strom zu liefern.
Also, neuer Tag, Tagesertrag wird auf null gesetzt. 🤡 

jetzt kommen die Fragen:
1. Ich wette, die Rheinfolge die du in der DTU hast ist: 
    Erste: HM1800
     Zweite: HM1000
Probiere die umkegehrte Rheinfolge (HM1000 erster, HM1800 zweiter)

2. Hast du vielleicht eine Zeitschaltung die den HMS1000 spaeter startet anstatt sofort (bevor die HMS1800 aufgehoert hat, Strom zu produzieren?). Bitte anpassen, damit keine "Leer-Pausen" existieren.

My 2cents und 🤞

**Comment by KEausW** [comment]
Die Wette hast du schonmal gewonnen ;-) Und ja ich habe den Marstek so konfiguriert das er 16.30 Uhr angeht.
An manchen Tagen ist da noch etwas vom HMS1800 da aber an trüben Tagen ist es da schon Stockdunkel.
Für was sind denn dann diese Einstellungen gut wenn die DTU den neuen Tag nach Hell/Dunkel bestimmt ?

![DTU01](https://github.com/user-attachments/assets/f81827d2-3a7a-422f-921c-ac272e4f1a17)

Grüße Karsten

**Reply by Manos1966** [reply]
Diese Einstellungen wurden gemacht weil wir Hoymiles angeschlossen an Batterien haben....
OpenDTU-**OnBattery**  😜 😜😜😜

Dann denkt die Hoymiles es ist immer Tag.

**Reply by Manos1966** [reply]
> ich habe den Marstek so konfiguriert das er 16.30 Uhr angeht.
An manchen Tagen ist da noch etwas vom HMS1800 da aber an trüben Tagen ist es da schon Stockdunkel.

Logisch! Vielleicht solltest du testen ob das Problem existiert, wenn die HMS800 staendig eingeschalten bleibt.

**Comment by KEausW** [comment]
Sorry, aber das verstehe ich nicht.
Warum gibt es diese Einstellungen dann auch in der normalen openDTU ohne Battery ?
Und wie kann ich den HMS1800 ständig laufen lassen wenn kein Strom da ist ? 
Oder meinst du den HMS1000 wo der Akku davor hängt ? Der läuft leider nur so lange
bis das der Akku auf 15% runter ist. Im Winter reicht das nicht sehr lang.
Grüße Karsten

**Reply by Manos1966** [reply]
Sorry, 
"Vielleicht solltest du testen ob das Problem existiert, wenn die HMS100 staendig eingeschalten bleibt." war gemeint.

> Der läuft leider nur so lange
bis das der Akku auf 15% runter ist. Im Winter reicht das nicht sehr lang.

Sie muss nicht produzieren. Sie muss nur online sein. 
Dies muss du aendern: "ich habe den Marstek so konfiguriert das er 16.30 Uhr angeht" auf so etwas wie 
"ich habe den Marstek so konfiguriert das er 13.30 Uhr angeht" und testen ob das Problem dadurch beseitigt wurde.

Dies ist meine Test HM1500 (siehe unten)
Die Pylontech ist unter 15%, daher hat der DPL die Stromproduktion abgeschaltet.
Trotzdem, die HM1500 ist online.


![image](https://github.com/user-attachments/assets/52212cc8-a603-4151-8f22-96f3e06ccbb6)
